<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ELIMINARCONDUCTOR</class>
 <widget class="QWidget" name="ELIMINARCONDUCTOR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelEliminar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ELIMINAR UNIDAD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bEliminarUnidad">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>ELIMINAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="lIDUnidad">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>IDUnidad</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboxIDUnidad">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bRegresarEUnidad">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>REGRESAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
